ui/raidboss/data/02-arr/trial/shiva-ex.txt

```diff
@@ -34,208 +34,208 @@
+1159.5 "Permafrost?" sync / 1[56]:Shiva:BE3:/
+1163.7 "Dreams Of Ice" sync / 1[56]:Shiva:BEA:/ window 20,20
 
-1166.9 "Frost Bow" sync /:Shiva:BDD:/ window 100,20 jump 942.8
-1172.0 "Glass Dance" #sync /:Shiva:BDF:/
-1194.6 "Avalanche" #sync /:Shiva:BE0:/
-1198.7 "Permafrost?" #sync /:Shiva:BE3:/
-1210.2 "Melt" #sync /:Shiva:C7E:/ window 20,20
-1212.4 "Dreams Of Ice" #sync /:Shiva:BEA:/
+1166.9 "Frost Bow" sync / 1[56]:Shiva:BDD:/ window 100,20 jump 942.8
+1172.0 "Glass Dance" #sync / 1[56]:Shiva:BDF:/
+1194.6 "Avalanche" #sync / 1[56]:Shiva:BE0:/
+1198.7 "Permafrost?" #sync / 1[56]:Shiva:BE3:/
+1210.2 "Melt" #sync / 1[56]:Shiva:C7E:/ window 20,20
+1212.4 "Dreams Of Ice" #sync / 1[56]:Shiva:BEA:/
 
 
 
 
 # Phase 7b: Sword
-1296.2 "Frost Blade" sync /:Shiva:993:/
-1298.5 "Icicle Impact (cross)" #sync /:Shiva:BEB:/
-1302.3 "Icebrand" sync /:Shiva:BE1:/
-1306.4 "Icicle Impact (cross)" #sync /:Shiva:BEB:/
-1306.4 "Permafrost?" sync /:Shiva:BE3:/
-1316.8 "--sync--" sync /:BE9:Shiva/ window 10,10
-1319.0 "Glacier Bash" sync /:Shiva:BE9:/
-1329.4 "Whiteout" sync /:Shiva:BEC:/
-1334.7 "Heavenly Strike" sync /:Shiva:BE8:/
-1338.8 "Permafrost?" sync /:Shiva:BE3:/
-1350.7 "Melt" sync /:Shiva:994:/ window 20,10
-1357.9 "Icicle Impact (circle)" duration 4 #sync /:Shiva:BEB:/
-1360.8 "Permafrost?" sync /:Shiva:BE3:/
-1365.0 "Dreams Of Ice" sync /:Shiva:BEA:/ window 20,20
-
-1368.2 "Frost Bow" sync /:Shiva:BDD:/ window 100,20 jump 942.8
-1373.3 "Glass Dance" #sync /:Shiva:BDF:/
-1395.9 "Avalanche" #sync /:Shiva:BE0:/
-1400.0 "Permafrost?" #sync /:Shiva:BE3:/
-1411.5 "Melt" #sync /:Shiva:C7E:/ window 20,20
-1413.7 "Dreams Of Ice" #sync /:Shiva:BEA:/
+1296.2 "Frost Blade" sync / 1[56]:Shiva:993:/
+1298.5 "Icicle Impact (cross)" #sync / 1[56]:Shiva:BEB:/
+1302.3 "Icebrand" sync / 1[56]:Shiva:BE1:/
+1306.4 "Icicle Impact (cross)" #sync / 1[56]:Shiva:BEB:/
+1306.4 "Permafrost?" sync / 1[56]:Shiva:BE3:/
+1316.8 "--sync--" sync / 14:Shiva:BE9:/ window 10,10
+1319.0 "Glacier Bash" sync / 1[56]:Shiva:BE9:/
+1329.4 "Whiteout" sync / 1[56]:Shiva:BEC:/
+1334.7 "Heavenly Strike" sync / 1[56]:Shiva:BE8:/
+1338.8 "Permafrost?" sync / 1[56]:Shiva:BE3:/
+1350.7 "Melt" sync / 1[56]:Shiva:994:/ window 20,10
+1357.9 "Icicle Impact (circle)" duration 4 #sync / 1[56]:Shiva:BEB:/
+1360.8 "Permafrost?" sync / 1[56]:Shiva:BE3:/
+1365.0 "Dreams Of Ice" sync / 1[56]:Shiva:BEA:/ window 20,20
+
+1368.2 "Frost Bow" sync / 1[56]:Shiva:BDD:/ window 100,20 jump 942.8
+1373.3 "Glass Dance" #sync / 1[56]:Shiva:BDF:/
+1395.9 "Avalanche" #sync / 1[56]:Shiva:BE0:/
+1400.0 "Permafrost?" #sync / 1[56]:Shiva:BE3:/
+1411.5 "Melt" #sync / 1[56]:Shiva:C7E:/ window 20,20
+1413.7 "Dreams Of Ice" #sync / 1[56]:Shiva:BEA:/
 
 
 
 # Enrage
 #1165.6 "--untargetable--"
 #1176.7 "--sync--" sync /:Shiva:C16:/
 #1176.7 "Diamond Dust" sync /:Shiva:9A2:/
-#1181.6 "Diamond Dust" sync /:Shiva:98A:/
\ No newline at end of file
+#1181.6 "Diamond Dust" sync /:Shiva:98A:/
```

Comment:
Why did this and the above two lines not get converted?

Reply:
Oops, yes. Most lines containing comments were already getting converted, but I neglected to add an exception for full-line comments. That's a simple enough change.